python/tvm/relay/transform/transform.py

```diff
@@ -666,7 +666,7 @@ def PartitionGraph():
     return _ffi_api.PartitionGraph()
 
 
-def AnnotateTarget(targets):
+def AnnotateTarget(targets, default_tuples=False):
```

Comment:
Perhaps `include_tuples=True` is more descriptive?

Reply:
refactored as include_non_call_ops, defaults to True